README.md

```diff
@@ -202,6 +206,22 @@ jobs:
           recreate: true 
 ```
 
+## Delete a comment which is no longer relevant
+
+Take a case where you need to delete a comment which is no longer relevant. E.g., we use this to write redirects for our Docs' site. If a contributor were to revert a filename change or a file deletion, we'd need to invalidate the comment. The `delete` flag will help in achieving this scenario.
```

Comment:
Our use-case example might be a bit advanced for beginners :sweat_smile: So maybe we make a simpler example like

```
Eg. Let us say we previously added a comment in a PR with `id: preview-url` to post a link where the changes of the pull request could be previewed. It might be useful to delete such a comment when the PR is closed to avoid users from accessing stale preview links
```